### DISCUSSION TITLE
How to use with .Net 5 and WinForms

### DISCUSSION THREAD

**Discussion by dbowser** [Live-Charts/LiveCharts2#168] [Q&A]
Hello,

I am trying to use this with .Net 5 and WinForms. 
Has anyone got this working? If so, can you give me any guidance? 
I would like to see the Sample app working but haven't been able to get that to build without errors. There seems to be a lot of dependency on .Net Framework still. 

Thanks

**Comment by beto-rodriguez** [comment]
My bad WinForms sample project was broken, it is working now since fc05052cbbe6d466c3d63a9f4538c43133616f15.
